postgres_exporter/common/queries_per_db.yml

```diff
@@ -63,7 +63,7 @@ ccp_stat_user_tables:
 
 
 ccp_table_size:
-  query: "SELECT current_database() as dbname, n.nspname as schemaname, c.relname, pg_total_relation_size(c.oid) as size_bytes FROM pg_catalog.pg_class c JOIN pg_catalog.pg_namespace n ON c.relnamespace = n.oid WHERE NOT pg_is_other_temp_schema(n.oid) AND relkind IN ('r', 'm', 'f')"
+  query: "SELECT current_database() as dbname, c.relnamespace::regnamespace as schemaname, c.relname, pg_total_relation_size(c.oid) as size_bytes FROM pg_catalog.pg_class WHERE NOT pg_is_other_temp_schema(c.relnamespace) AND relkind IN ('r', 'm', 'f')"
```

Comment:
The alias for pg_class was removed but not from the column names. Can either add the alias back or remove the `c.` prefix. I'm fine either way.